supervisor/docker/stats.py

```diff
@@ -14,9 +14,12 @@ def __init__(self, stats):
         self._blk_write = 0
 
         try:
-            self._memory_usage = (
-                stats["memory_stats"]["usage"] - stats["memory_stats"]["stats"]["cache"]
-            )
+            if "total_inactive_file" in stats["memory_stats"]["stats"]:
+                cache = stats["memory_stats"]["stats"]["total_inactive_file"]
+            else:
+                cache = stats["memory_stats"]["stats"]["cache"]
+
+            self._memory_usage = stats["memory_stats"]["usage"] - cache
```

Comment:
```suggestion
             self._memory_usage = stats["memory_stats"]["usage"] - stats["memory_stats"]["stats"].get("total_inactive_file", "cache")
```
Simplified variant

Reply:
But that would return `"cache"` if `total_inactive_file` doesn't exist:

```
>>> myd1 = { "total_inactive_file": 1, "cache": 2 }
>>> myd2 = { "cache": 3 }
>>> myd1.get("total_inactive_file", "cache")
1
>>> myd2.get("total_inactive_file", "cache")
'cache'
```

Reply:
Right...